### PR TITLE
Update rules for capitalizing domain objects.

### PR DESCRIPTION
**Pull request by jaywood128**
Update rules for capitalizing domain objects with examples

### CODE REVIEW

**Review by robotdan — APPROVED** [review]

**Review by mooreds — CHANGES_REQUESTED** [review]
Please add a note on the welcome page explaining the convention (can be similar text thw ath you have in the examples). https://github.com/FusionAuth/fusionauth-site/blob/master/site/docs/v1/tech/index.adoc

**Comment by jaywood128** [comment]
> https://github.com/FusionAuth/fusionauth-site/blob/master/site/docs/v1/tech/index.adoc

The index page is customer-facing so why would be include our internal style guidelines there? Is this if someone wanted to open a PR against fusionauth-site? @mooreds

**Comment by mooreds** [comment]
So our customers know what our convention is. They're the ones reading it, after all :) @jaywood128

File: site/docs/v1/tech/index.adoc [hunk 1]
```diff
@@ -35,5 +35,12 @@ Here is an overview of each of the sections:
 * *Premium Features*: FusionAuth has a full featured community edition, but some features are reserved for customer with a license key. This section outlines the features and how to use them.
 * *APIs*: APIs are the heart of FusionAuth and are documented in depth in this section.
 * *Release Notes*: The all important release notes so you can stay current on what is new in FusionAuth!
+* *Documentation Style Guidelines*: Here are some guidelines to follow when contributing documentation.
```
Comment (mooreds) [inline site/docs/v1/tech/index.adoc:38]:
```suggestion

== Documentation Conventions

```

File: site/docs/v1/tech/index.adoc [hunk 2]
```diff
@@ -35,5 +35,12 @@ Here is an overview of each of the sections:
...
+- Capitalize all domain objects, especially when calling the API in which the object is created and/or updated in FusionAuth.
```
Comment (mooreds) [inline site/docs/v1/tech/index.adoc:39]:
```suggestion
- We apitalize all domain objects, especially when calling the API in which the object is created and/or updated in FusionAuth.
```

File: site/docs/v1/tech/index.adoc [hunk 3]
```diff
@@ -35,5 +35,12 @@ Here is an overview of each of the sections:
...
+For example, see the link:/docs/v1/tech/apis/api-keys[API Key API] description for `apiKeyId`, where API Key is capitalized: `The unique Id of the API Key to create. If not specified a secure random UUID will be generated.`
+- If referring to something that exists as a domain object in FusionAuth, but you are not explicitly referring to an object being created/updated in FusionAuth, use lowercase.
```
Comment (mooreds) [inline site/docs/v1/tech/index.adoc:41]:
```suggestion
- If referring to something which is a domain object in FusionAuth, but we are not explicitly referring to an object being created/updated in FusionAuth, we use lowercase.
```

**Review by mooreds — CHANGES_REQUESTED** [review]
Modified to reflect that this is guidance for a user reading, not a contributor.

**Pull request merged**